frameworks/flaml/__init__.py

```diff
@@ -0,0 +1,42 @@
+from amlb.resources import config as rconfig
+from amlb.utils import call_script_in_same_dir
+
+def setup(*args, **kwargs):
+    call_script_in_same_dir(__file__, "setup.sh", rconfig().root_dir, *args, **kwargs)
+
+
+def run(dataset, config):
+    from frameworks.shared.caller import run_in_venv
+    enc = config.framework_params.get('_enc', False)
+    if enc:
+        from amlb.datautils import impute
+        X_train_enc, X_test_enc = impute(dataset.train.X_enc, dataset.test.X_enc)
+        data = dict(
+            train=dict(
+                X_enc=X_train_enc,
+                y_enc=dataset.train.y_enc
+            ),
+
+            test=dict(
+                X_enc=X_test_enc,
+                y_enc=dataset.test.y_enc
+            ),
+
+            problem_type=dataset.type.name  
+        )
```

Comment:
Do you still need this or were you just testing?
The encoded data should only be used by frameworks that only support numeric data.

Reply:
removed. Thanks!